### ISSUE TITLE
Solarized Light theme highlighting issue

### ISSUE THREAD

**Issue by GeorgeWells** [Microsoft/vscode#24040]
- VSCode Version: Code 1.11.0 (9122ec4b1068bd391e41974f46a4b54c1b9c90f9, 2017-04-05T20:59:05.375Z)
- OS Version: Windows_NT ia32 6.1.7601
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|githistory|donjayamanne|0.2.0|
|php-debug|felixfbecker|1.10.0|
|php-intellisense|felixfbecker|1.1.4|

Highlighting text on Solarized Light theme is very dark - same colour as the colour used for key words in .php and .js files 'class', 'function' etc. which makes them invisible.

Steps to Reproduce:

1. Select solarized light theme
2. Highlight stuff

![image](https://cloud.githubusercontent.com/assets/23452044/24751517/cc9be092-1ac1-11e7-873a-62e7339658e0.png)

![image](https://cloud.githubusercontent.com/assets/23452044/24751543/e70ac5ba-1ac1-11e7-9b4e-f98df1511998.png)

**Comment by jurosh** [comment]
I've got another but probably related issue with Solarized Light theme:
- in Json files, it's automatically selecting whole words - instead just blinking cursor
- see: ![screen](http://screen.jurosh.com/i15/2017_04_06_32560309962311719044.png)

**Comment by GeorgeWells** [comment]
I believe it's doing what it should be doing (click on a word and it highlights all instances). However it should be a fairly light highlight as opposed to quite dark.

**Comment by GeorgeWells** [comment]
As you can see by this screenshot, it should be fairly light (Light+ theme).

![image](https://cloud.githubusercontent.com/assets/23452044/24752050/197f68be-1ac4-11e7-8601-bf0162a7d45c.png)

**Comment by benesch** [comment]
This same issue has been driving me crazy. Thanks for reporting, @GeorgeWells.

As best as I can tell, we were spoiled in 1.10 with a light selection color (#EEE8D5) thanks to ee8bce102195fccc84ad53b999ccba6374bb9b93. As it turns out, this was an accident, and was reverted to the dark color (#073642) for 1.11. Using the dark color for selections is how Ethan Schoonover originally designed it as well ([source](https://github.com/altercation/ethanschoonover.com/blob/master/projects/solarized/textmate-colors-solarized/Solarized%20(light).tmTheme#L23)).

I think solving this issue will require using the new workbench color customizations to set a different color for the `editor.selectionHighlight`'s highlight. For now, though, setting `"editor.selectionHighlight": false` in your settings will disable the click-on-word-and-highlight-all-instances that's particularly distracting with this selection color.

**Comment by benesch** [comment]
Actually, here's an even more surgical approach. Drop the following block into your preferences

```json
    "workbench.experimental.colorCustomizations": {
        "editorSelectionHighlight": "#E0DBC9"
    }
```

which will color the words that match the one you've clicked on with a gray that's just slightly darker than the line highlight (#E0DBC9), while leaving other selections the default dark blue (#073642).

**Comment by aeschli** [comment]
fixed by #24292
